source/_integrations/insteon.markdown

```diff
@@ -49,51 +49,82 @@ Overview of supported Insteon modems & hubs
+
+The following capabilities are available in the Insteon configuration panel:
+
+### Add device
+
+In order for any two Insteon devices to talk with one another, they must be linked. For an overview of device linking, please read the Insteon page on [understanding linking]. To add an Insteon device:
+
+1. Open the [Insteon configuration panel](#insteon-configuration-panel).
+2. Press the **Add device** button on the bottom right corner of the screen.
+3. Fill in the **Add device** dialog using the following options:
+
+- **Device address**: Enter the Insteon address of the device, such as `1A.2B.3C`, to link a specific device remotely. Not all devices support remote linking. If a device does not support remote linking, press the `Set` button on the device to put the device in linking mode.
+- **Link multiple**: To add multiple devices in a single session, select the **Add multiple** checkbox.
+
+### Managing Insteon scenes
+
+Insteon scenes can be created, changed or deleted using the **Scenes** tab of the [Insteon configuration panel](#insteon-configuration-panel). To trigger an Insteon scene see [Triggering Insteon Scenes](#triggering-insteon-scenes) below.
+
+### Device properties
+
+Insteon device properties, such as the LED brightness, can be managed using the Insteon configuration panel. To see the available properties of a device, select the device from the list of devices in the Insteon configuration panel. This will display the list of available properties for the specific device on the **Properties** tab. Each device type will have a different set of properties and not all devices have properties.
+
+- **Read device properties**:  Use the **Load from device** menu option to read the properties from the device.
+- **Change device properties**: Select the specific property from the list of properties. Use the dialog box to edit the property value. This does not write the change to the device.
+- **Write property changes**: Use the **Write to device** menu option to save the property changes to the device.
+- **Undo changes**: Property changes can be undone before they are written to the device by selecting the **Undo changes** menu option.
+- **Delete device**: Deletes the Insteon device from Home Assistant and removes all references to the device in the modem. Optionally, it can remove any references to the device in other Insteon devices.
+
+### Device All-Link Database
+
+The Insteon All-Link Database (ALDB) contains the list of links to other devices in the Insteon network. The All-Link Database can be managed using the Insteon configuration panel. To see the All-Link Database of a device, select the device from the list of devices in the Insteon configuration panel and select the **All-Link Database** tab.
+
+- **Read the ALDB**: Use the **Load from device** menu option to read the current ALDB records from the device.
```

Comment:
**Read the ALDB**: is this a UI string?

Reply:
"Load from device" is the UI string which results in reading the All-Link Database (ALDB)